Xamarin.Forms.Platform.iOS/Renderers/ShellSectionRenderer.cs

```diff
@@ -307,7 +323,7 @@ protected virtual void OnInsertRequested(NavigationRequestedEventArgs e)
 
 			_trackers[page] = tracker;
 
-			ViewControllers.Insert(ViewControllers.IndexOf(beforeRenderer.ViewController), renderer.ViewController);
```

Comment:
We are now setting `ViewControllers` array instead of adding something to existing collection, will that screww any type of animation ?

Reply:
This code only runs when you're inserting a page not navigating to it. 

The `InsertPageBefore` only lets you insert pages behind the currently visible page and doesn't support any type of animation.  This code doesn't effect Pushing pages only inserting